packages/firestore/CHANGELOG.md

```diff
@@ -1,4 +1,10 @@
-# Unreleased
+# 0.7.3 (Unreleased)
+- [fixed] Fixed an issue where the first `get()` call made after being offline
+  could incorrectly return cached data without attempting to reach the backend.
+- [changed] Changed `get()` to only make 1 attempt to reach the backend before
+  returning cached data, potentially reducing delays while offline.
```

Comment:
Is it worth noting here that the two attempts mechanism was a work around for a specific backend bug that's no longer in effect?